scripts/pt/portugal_geoedif.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os.path
+
+# The source data can be obtained by running
+# esri2geojson https://inenetw05.ine.pt:6443/arcgis/rest/services/GeoEdif/GEOEDIF_AC25/MapServer/0 pt-azores25.json
+# esri2geojson https://inenetw05.ine.pt:6443/arcgis/rest/services/GeoEdif/GEOEDIF_AC26/MapServer/0 pt-azores26.json
+# esri2geojson https://inenetw05.ine.pt:6443/arcgis/rest/services/GeoEdif/GEOEDIF_MAD/MapServer/0 pt-madeira.json
+# esri2geojson https://inenetw05.ine.pt:6443/arcgis/rest/services/GeoEdif/GEOEDIF_CONT/MapServer/0 pt-continente.json
+# In practice the continental dataset is too large (>3M points), so the server fails before returning the count. Forcing esri2geojson to use OIDs does the trick
```

Comment:
The server failing to return count is often an issue even with the web interface, not just with pyesridump. Solved it by just commenting out the test for pagination support in the dumper and thus forcing OIDs.